angular-client/package.json

```diff
@@ -14,7 +14,6 @@
   "private": true,
   "dependencies": {
     "@angular-devkit/build-angular": "^18.2.4",
-    "@angular/animations": "^18.2.4",
```

Comment:
we don't want to remove this, you should undelete this line and rollback the changes to the package-lock.json